server/controllers/stock/reports/stock/value.js

```diff
@@ -30,21 +30,109 @@ async function reporting(_options, session) {
 
   const report = new ReportManager(STOCK_VALUE_REPORT_TEMPLATE, session, optionReport);
 
+
   const options = (typeof (_options.params) === 'string') ? JSON.parse(_options.params) : _options.params;
   data.dateTo = options.dateTo;
   data.depot = await db.one('SELECT * FROM depot WHERE uuid=?', [db.bid(options.depot_uuid)]);
-  const stockValues = await db.exec('CALL stockValue(?,?,?);', [
```

Comment:
If we are not longer using the `stockValue()` procedure, let's remove it from the stored procedures.